pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
whatever we do here we also need to do in generic/rest/proxy.go (a refactor to make apiserver/proxy.go delegate to the generic proxy is overdue, but should be a separate PR)